### ISSUE TITLE
Looking for quick funds

### ISSUE THREAD

**Issue by sadrev25** [Morningstar88/X#14]
## Ask mike
     Hey Mike! we have a software project here, Do you have any idea where we rise funds? Do you know any groups, maybe lion clubs?
##Lions clubs
  Hi are you open on Thursday, we would like to visit for coffee
###To do jake.take out the pictures on GitHub #emergency

**Comment by Morningstar88** [comment]
Lion's club meet 1800 on Thursday....Message ID: ***@***.***>
Lets go. 

Meet at 430 by the temple...

**Comment by Morningstar88** [comment]
I will do so tomorrow, in case he's asleep...
You can remind me in the morning. 

I will do my best to fix the app too... I have a task list. 

Lets meet online tomorrow, about 10 or 11pm, to test the app and get more feedback.

**Comment by sadrev25** [comment]
Okay sure jake

**Comment by sadrev25** [comment]
Hi jake,Did you contacted mike?

**Comment by sadrev25** [comment]
Hi jake,my mobile no+886 979 587806, i will start by 7 today and jake bring guitar , it help ful for raise fund

**Comment by Morningstar88** [comment]
Guitar is a good idea!...

See you 8pm, Ximending, Exit 1. 

My phone is off, so I cannot confirm. just see you there. 
Our job today:

- Take many photos and videos of coin on your phone. 
- First trial. 

My friend's number is here, if you need anything:

Tyler Number

0909570784

*

See you then!

**Comment by Morningstar88** [comment]
Hi,

The coins are not quite ready yet.

We made some basic wood prototypes, but not sure if its good enough yet.

Can we meet later, or another day?

Let me know. 

I tried to call you. 

Best Wishes,

Jake

-- 
Ask yourself this question: 'Will this matter a year from now?'



Jan 7, 2023, 06:58 by ***@***.***:

**Comment by sadrev25** [comment]
Sure jake! No problem

**Comment by sadrev25** [comment]
Sure jake!No problem,will meet.
Cheers

**Comment by sadrev25** [comment]
Hi jake, will meet today evening?I had a new idea, need to discuss with you and will have weekend meeting.
Haha
Cheers

**Comment by Morningstar88** [comment]
Hello!

Sorry, Im a little tired today, but Tuesday is good, if you're free. 
 
Please tell me a little about the idea now, we can discuss it online and move forward.

I was thinking of making coins with passive RFID tags, like Taipei MRT tokens, but as money.

lLt me know your idea if you like.

**Comment by Morningstar88** [comment]
Hi,

I'm not sure if they're open tomorrow. 

Seems just a year end party.

Not sure what to do..What do you think?
https://www.lions300a2.org/a2/109310/index/zonearea/googlecalendaragenda.html

**Comment by sadrev25** [comment]
We can try or else will try see any other club beacuse if Chinese new year started everything it will close.or will have meet tomorrow and will decide.

**Comment by sadrev25** [comment]
sure jake, will discuss about new ideas and fundraising tomorrow,

**Comment by Morningstar88** [comment]
Yes, we also need to research where to go, if we meet on Wednesday...

See you 430pm today!

**Comment by Morningstar88** [comment]
We can ask this bot about fundraising...

https://beta.character.ai/chat?char=pDuXfmuMhtjMSkJAObMfTbH-qD2yeBIppc2vIZLjZss

It is very intelligent.

please bookmark it. See you 430pm!

**Comment by sadrev25** [comment]
will speak about this in today's meeting jake!
Cheers!!!

**Comment by Morningstar88** [comment]
hello. lets meet at 5pm. is that alright?

thanks
-- 
Ask yourself this question: 'Will this matter a year from now?'



Jan 17, 2023, 15:44 by ***@***.***:

**Comment by sadrev25** [comment]
yeah sure jake

**Comment by Morningstar88** [comment]
Hi, please tell me 5 minutes before you arrive.

**Comment by Morningstar88** [comment]
everything ok, what time will you be here?Message ID: ***@***.***>

**Comment by Morningstar88** [comment]
hi, are you here?
-- 
Ask yourself this question: 'Will this matter a year from now?'



Jan 17, 2023, 16:08 by ***@***.***:

**Comment by Morningstar88** [comment]
Good morning, I started a new issue last night.

Working online all day today, til our meeting. 

lets build!
-- 
Ask yourself this question: 'Will this matter a year from now?'



Jan 17, 2023, 16:08 by ***@***.***:

**Comment by Morningstar88** [comment]
Hi, are we meeting today?

**Comment by sadrev25** [comment]
Sure jake

**Comment by sadrev25** [comment]
At what time ?

**Comment by Morningstar88** [comment]
Message ID: ***@***.***>Can meet any time you want.

Sorry, my mail didn't send...

**Comment by sadrev25** [comment]
Will meet tomorrow jake

**Comment by Morningstar88** [comment]
ok,,, sorry I missed you... is 430pm good?

See you then!!!
-- 
Ask yourself this question: 'Will this matter a year from now?'



Jan 18, 2023, 17:31 by ***@***.***:
